mvrx/src/main/kotlin/com/airbnb/mvrx/MvRxView.kt

```diff
@@ -23,6 +23,15 @@ private val handler = Handler(Looper.getMainLooper(), Handler.Callback { message
  * will automatically subscribe to all state changes in the ViewModel and call [invalidate].
  */
 interface MvRxView : MvRxViewModelStoreOwner, LifecycleOwner {
+
+    /**
+     * A globally unique id for this MvRxView. If your MvRxView is being recreated due to a lifecycle event (e.g. rotation)
+     * you should assign a consistent id. Likely this means you should save the id in onSaveInstance state.
+     * The viewId will not be accessed until a subscribe method is called. Accessing mvrxViewId before calling
+     * super.onCreate() will cause a crash.
```

Comment:
You might want to give an example implementation of this for people who are only looking at this and are implementing MvRxView themself.